airflow/sensors/external_task.py

```diff
@@ -351,7 +352,7 @@ def execute_complete(self, context, event=None):
             self.log.info("External task %s has executed successfully.", self.external_task_id)
             return None
         elif event["status"] == "timeout":
-            raise AirflowException("Dag was not started within 1 minute, assuming fail.")
+            raise AirflowException("Dag was not started within timeout, assuming fail.")
```

Comment:
Is it possible to show the timeout used in the message? I feel this might help debugging.

Reply:
changed text to f-string with self.timeout